src/gui/SendButtonIndicator.cpp

```diff
@@ -2,32 +2,30 @@
 
 #include "embed.h"
 #include "Mixer.h"
-#include "MixerLine.h"
+#include "MixerChannelView.h"
 #include "MixerView.h"
 
 namespace lmms::gui
 {
 
-SendButtonIndicator:: SendButtonIndicator( QWidget * _parent, MixerLine * _owner,
-										   MixerView * _mv) :
-	QLabel( _parent ),
-	m_parent( _owner ),
-	m_mv( _mv )
+SendButtonIndicator:: SendButtonIndicator(QWidget * parent, MixerChannelView * owner, MixerView * mv) :
```

Comment:
Extra space between types and `*`

Reply:
Fixed with commit 92878ce4030. Using reference/pointer at type like in other places in the code. Underscore was removed.